src/Web/Telegram/API/Bot/API.hs

```diff
@@ -5,7 +5,10 @@
 
 module Web.Telegram.API.Bot.API
   ( -- * Functions
-    getMe
+    runClient
+  , runClientEx
```

Comment:
1. Servant's API is more convenient, when you need to provide all parameters in a reader.
2. I want my program (and your library, maybe) to be testable against mock Telegram server.